doc/user/layouts/shortcodes/tabs.html

```diff
@@ -7,11 +7,18 @@
 <script>
     $(document).ready(function () {
 
-    $('.tab-content').find('.tab-pane').each(function (idx, item) {
-        var navTabs = $(this).closest('.code-tabs').find('.nav-tabs'),
-            title = $(this).attr('title');
-        navTabs.append('<li><a href="#">' + title + '</a></li');
-    });
+      $(this).find('.code-tabs').each(function (idx, item) {
+        // For each tabs group; make new unordered list
+        var n = $("<ul/>", {
+          class: "nav nav-tabs"
+        });
+        // For each tab pane; make new list item
+          $(this).find('.tab-pane').each(function (idx, item) {
+          n.append('<li><a href="#">' + $(this).attr("title") + '</a></li>');
+        })
+        // Replace
+        $(this).find('.nav-tabs').replaceWith(n);
+      });
```

Comment:
Ah, ok, so this change makes initializing tabs twice idempotent? I think the root cause here is actually that we are installing the tab initialization code in every `{{< tab >}}` shortcode, but the initialization code applies to _all_ tab groups.

I propose that we move this entire `<script>` block out of `tabs.html` and into `head.html`. Then it will only run once, and you won't need to make it idempotent. (You could stick with the new idempotent implementation, if you like, but it will no longer be necessary.)